functions/fish_right_prompt.fish

```diff
@@ -11,6 +11,7 @@
 set -q color_vi_mode_indicator; or set color_vi_mode_indicator black
 set -q color_vi_mode_normal; or set color_vi_mode_normal green
 set -q color_vi_mode_insert; or set color_vi_mode_insert blue 
+set -q color_vi_mode_replace; or set color_vi_mode_replace $color_vi_mode_insert 
```

Comment:
The default value of `color_vi_mode_replace` is to be determined.

It is temporarily set to `$color_vi_mode_insert` as in https://github.com/oh-my-fish/theme-bobthefish/blob/954fbc961b779c29ad24749f9a09095d049fb752/functions/fish_mode_prompt.fish#L34